sympy/functions/combinatorial/factorials.py

```diff
@@ -363,6 +363,16 @@ def _eval_is_integer(self):
         if self.args[0].is_integer and self.args[0].is_nonnegative:
             return True
 
+    def _eval_rewrite_as_factorial(self, arg, **kwargs):
+        from sympy import summation
+        i = Dummy('i')
+        f = S.NegativeOne**i / factorial(i)
+        return factorial(arg) * summation(f, (i, 0, arg))
+
+    def _eval_rewrite_as_gamma(self, arg, **kwargs):
+        from sympy import exp, exp_polar, gamma, I, lowergamma
+        return ((-1)**(arg + 1)*exp(-I*pi*arg)*lowergamma(arg + 1, exp_polar(I*pi)) + gamma(arg + 1))*exp(-1)
```

Comment:
It looks like `exp_polar(I*pi)` could be `-1`. Then `exp_polar` need not be imported.

Reply:
Made the changes.